### PR TITLE
Fix release workflow by fix path to release docs

### PR DESCRIPTION
**Pull request by Czaki**
# Description

Fix path to release notes, add printing some values to better understand problems in case of failure.

### CODE REVIEW

File: .github/workflows/make_release.yml [hunk 1]
```diff
@@ -41,13 +44,20 @@ jobs:
...
 
+      - name: Debug print
+        run: |
+          echo "tag=${TAG}"
+          echo "release_notes_path=${RELEASE_NOTES_PATH}"
+          ls -l docs/docs/release
+
       - name: Create Release
         uses: "softprops/action-gh-release@v2"
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
```
Comment (jni) [inline .github/workflows/make_release.yml:57]: Clever, with the new triggers!

Comment (jni) [reply .github/workflows/make_release.yml:57]: ... But you need the same guard for the PyPA push!!!

Comment (Czaki) [reply .github/workflows/make_release.yml:57]: Trusted publisher saved us

**Review by jni — APPROVED** [review]
Looks like it worked! 🥳

**Comment by jni** [comment]
... Well, the ver is clearly wrong:

release_notes_path: docs/docs/release/release_refs/pull/7082/merge.md

😅 But I don't think there's anything we can do about it in a PR?

**Comment by Czaki** [comment]
> ... Well, the ver is clearly wrong:
> 
> release_notes_path: docs/docs/release/release_refs/pull/7082/merge.md

The problem is about ref: `refs/pull/7082/merge`. We may get tag from napari itself (we install it before) but it should be only a problem for debug, not tagged commit.

**Comment by jni** [comment]
Do you think this is ready to merge then @Czaki? (I do.)

**Comment by Czaki** [comment]
Possible way to determine tag from napari installation `python -c "from importlib.metadata import version;from packaging.version import parse;print(parse(version('napari')).base_version)"`

**Comment by jni** [comment]
Random idea from seeing a bunch of yellow here: put a guard in the test_pull_request workflows that won't run the tests if unrelated workflows are changed?

**Pull request merged**